tools/tools.go

```diff
@@ -1,4 +1,4 @@
-// +build tools
+//go:build tools
```

Comment:
This line fixes a goimports error:
https://app.circleci.com/pipelines/github/crowdworks/slacts/115/workflows/aeb2f79d-c22e-420e-9d8c-d111d2350647/jobs/148

It's a new build constraint introduced in Go1.17.
https://go.dev/doc/go1.17

//go:build lines
The go command now understands //go:build lines and prefers them over // +build lines. The new syntax uses boolean expressions, just like Go, and should be less error-prone. As of this release, the new syntax is fully supported, and all Go files should be updated to have both forms with the same meaning. To aid in migration, gofmt now automatically synchronizes the two forms. For more details on the syntax and migration plan, see https://golang.org/design/draft-gobuild.